app/models/reservation_scopes.rb

```diff
@@ -50,6 +50,9 @@ def self.included(base) # rubocop:disable MethodLength, AbcSize
       scope :checkoutable, lambda {
         where('start_date <= ?', Time.zone.today).reserved
       }
+      scope :future, lambda {
```

Comment:
It should include all checkoutable reservations, correct? That's how I have it now.